legacy/src/app/partials/dhcp-snippets-table.html

```diff
@@ -192,11 +192,11 @@ <h2 data-ng-click="snippetAddCancel()" class="u-float--left">Adding snippet</h2>
 </div>
 <ul class="p-inline-list--middot">
   <li class="p-inline-list__item" data-ng-if="!hideAllSnippetsLink">
-    <a href="/#/settings/dhcp">All snippets: Settings > DHCP snippets</a>
+    <a href="/MAAS/r/settings/dhcp">All snippets: Settings > DHCP snippets</a>
```

Comment:
Is it possible to use the env vars here to add the "/MAAS" and "/r" parts? In ./legacy/.env there should be two variables:
```
BASENAME="/MAAS"
REACT_BASENAME="/r"
```

Reply:
@huwshimi No, they don't work in the HTML templates

Reply:
Nevermind - I found a way :)

Reply:
Thanks for figuring that out! Looks great.